<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="10" passed="7" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-02-05T23:27:09 IST" name="Suite" finished-at="2023-02-05T23:27:30 IST" duration-ms="20520">
    <groups>
    </groups>
    <test started-at="2023-02-05T23:27:09 IST" name="Test" finished-at="2023-02-05T23:27:30 IST" duration-ms="20520">
      <class name="com.test.DDTests">
        <test-method signature="testPostuser(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:1, instance:com.test.DDTests@6379eb]" started-at="2023-02-05T23:27:11 IST" name="testPostuser" data-provider="Data" finished-at="2023-02-05T23:27:15 IST" duration-ms="3338" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[10101]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[testuser11]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[John]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Canedy]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[a@gmail.com]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[test@1]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[1234567897]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPostuser -->
        <test-method signature="testPostuser(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:1, instance:com.test.DDTests@6379eb]" started-at="2023-02-05T23:27:15 IST" name="testPostuser" data-provider="Data" finished-at="2023-02-05T23:27:16 IST" duration-ms="1580" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[10201]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[testuser21]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Kim]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Kom]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[b@gmail.com]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[test@2]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[1234567897]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPostuser -->
        <test-method signature="testPostuser(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:1, instance:com.test.DDTests@6379eb]" started-at="2023-02-05T23:27:16 IST" name="testPostuser" data-provider="Data" finished-at="2023-02-05T23:27:18 IST" duration-ms="1406" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[10301]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[testuser31]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Smith]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Steve]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[c@gmail.com]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[test@3]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[1234567897]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPostuser -->
        <test-method signature="testDeleteUserByName(java.lang.String)[pri:2, instance:com.test.DDTests@6379eb]" started-at="2023-02-05T23:27:18 IST" name="testDeleteUserByName" data-provider="UserNames" finished-at="2023-02-05T23:27:19 IST" duration-ms="1581" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[testuser11]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1418)
at org.testng.Assert.assertEquals(Assert.java:1382)
at org.testng.Assert.assertEquals(Assert.java:1428)
at com.test.DDTests.testDeleteUserByName(DDTests.java:36)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUserByName -->
        <test-method signature="testDeleteUserByName(java.lang.String)[pri:2, instance:com.test.DDTests@6379eb]" started-at="2023-02-05T23:27:19 IST" name="testDeleteUserByName" data-provider="UserNames" finished-at="2023-02-05T23:27:21 IST" duration-ms="1470" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[testuser21]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1418)
at org.testng.Assert.assertEquals(Assert.java:1382)
at org.testng.Assert.assertEquals(Assert.java:1428)
at com.test.DDTests.testDeleteUserByName(DDTests.java:36)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUserByName -->
        <test-method signature="testDeleteUserByName(java.lang.String)[pri:2, instance:com.test.DDTests@6379eb]" started-at="2023-02-05T23:27:21 IST" name="testDeleteUserByName" data-provider="UserNames" finished-at="2023-02-05T23:27:22 IST" duration-ms="1592" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[testuser31]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1418)
at org.testng.Assert.assertEquals(Assert.java:1382)
at org.testng.Assert.assertEquals(Assert.java:1428)
at com.test.DDTests.testDeleteUserByName(DDTests.java:36)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUserByName -->
      </class> <!-- com.test.DDTests -->
      <class name="com.test.TestUser">
        <test-method is-config="true" signature="setUpData()[pri:0, instance:com.test.TestUser@49139829]" started-at="2023-02-05T23:27:22 IST" name="setUpData" finished-at="2023-02-05T23:27:22 IST" duration-ms="154" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpData -->
        <test-method signature="testCreateUser()[pri:1, instance:com.test.TestUser@49139829]" started-at="2023-02-05T23:27:22 IST" name="testCreateUser" finished-at="2023-02-05T23:27:24 IST" duration-ms="1545" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateUser -->
        <test-method signature="testGetUser()[pri:2, instance:com.test.TestUser@49139829]" started-at="2023-02-05T23:27:24 IST" name="testGetUser" finished-at="2023-02-05T23:27:25 IST" duration-ms="1366" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetUser -->
        <test-method signature="testUpdateUser()[pri:3, instance:com.test.TestUser@49139829]" started-at="2023-02-05T23:27:25 IST" name="testUpdateUser" finished-at="2023-02-05T23:27:28 IST" duration-ms="2860" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUpdateUser -->
        <test-method signature="testDeleteUser()[pri:4, instance:com.test.TestUser@49139829]" started-at="2023-02-05T23:27:28 IST" name="testDeleteUser" finished-at="2023-02-05T23:27:30 IST" duration-ms="1355" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUser -->
      </class> <!-- com.test.TestUser -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
